bootstrapvz/plugins/puppet/example/jessie-virtio-with-puppet.yaml

```diff
@@ -0,0 +1,45 @@
+---
```

Comment:
You should put that in `manifests/examples/kvm/jessie-puppet.yml`. When building the python package this folder wont be included.